sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -752,6 +753,44 @@ def test_radius_neighbors_factory_method_wrong_usages():
         )
 
```

Comment:
Can you introduce `test_pairwise_distances_is_usable_for` to assert that `PairwiseDistances` is usable?

Also, I think we might want to extend the parametrization and the body of those tests for `PairwiseDistances`:

 - `test_format_agnosticism`
 - `test_n_threads_agnosticism`
 - `test_strategies_consistency`
 - `test_memmap_backed_data`

What do you think?

Reply:
At first sight, It seems better to add new methods than to try to add `pytest.mark` parameters, since `ArgKmin` and `RadiusNeighbors` methods have different signatures than PairwiseDistances.